packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_model.dart

```diff
@@ -134,13 +478,24 @@ class LoggingTableModel extends ChangeNotifier with DisposerMixin {
     throw Exception('Implement this when needed');
   }
 
+  double getLogHeight(int index) {
+    final entry = _logs.elementAt(index);
+    final cachedHeight = entry.height;
```

Comment:
I don't think these two lines are necessary. If `entry.height != null`, the conditional assignment below doesn't happen and `entry.height` is still returned.